fvm/environment/value_store.go

```diff
@@ -153,7 +153,16 @@ func (store *valueStore) SetValue(
 		return errors.NewInvalidInternalStateAccessError(id, "modify")
 	}
 
-	err := store.meter.MeterComputation(
+	oldValue, err := store.GetValue(owner, keyBytes)
+	if err != nil {
+		return fmt.Errorf("get value failed: %w", err)
```

Comment:
Do we need to check if the error is not found? 

Can we add a comment and test case for it?

Reply:
Looks like it returns `nil, nil` [when key does not exist](https://github.com/onflow/flow-go/blob/master/fvm/storage/state/storage_state.go#L75)